src/storage.ts

```diff
@@ -291,9 +296,10 @@ export class Storage extends Service {
    */
   constructor(options: StorageOptions = {}) {
     options.apiEndpoint = options.apiEndpoint || 'www.googleapis.com';
+    options.apiScheme = options.apiScheme || 'https';
     const config = {
       apiEndpoint: options.apiEndpoint,
-      baseUrl: `https://${options.apiEndpoint}/storage/v1`,
+      baseUrl: `${options.apiScheme}://${options.apiEndpoint}/storage/v1`,
```

Comment:
What is an example of a time I *wouldn't* want to use https?

Reply:
Connecting to a local emulator.

Reply:
That makes a lot of sense.  I think we want to . rely on on an env var for that anyway (see above)